.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: Test
+on:
+  push:
+    branches: [main]
+    paths: [src/**, action.yml, .github/wiki/**, .github/workflows/test.yml]
+  pull_request:
+    branches: [main]
+    paths: [src/**, action.yml, .github/wiki/**, .github/workflows/test.yml]
+# We don't want cancel-in-progress because we want each test (if there are
+# multiple at a given time) to still attempt to push to the wiki. This should
+# not cause any hiccups on the wiki since any and all tests are publishing in
+# the same content.
+concurrency: test-${{ github.ref }}
+permissions:
+  contents: write
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./
+        with:
+          # For now, you'll need to manually specify a GitHub token until we
+          # solve #2. The x: prefix is a dummy username.
+          token: x:${{ secrets.GITHUB_TOKEN }}
```

Comment:
Want to switch to `${{ github.token }}`? I think it's just slightly more obvious that it's the built-in token this way.

Reply:
secrets.GITHUB_TOKEN is wayyyy more popular: 11x to be exact. imo following convention is better in this case.

https://github.com/search?q=secrets.GITHUB_TOKEN&type=code https://github.com/search?q=github.token&type=code

Reply:
Yeah, I do use `secrets.GITHUB_TOKEN` everywhere, too. I guess I'm thinking that `secrets.GITHUB_TOKEN` could be overridden by explicitly setting it as an actions secret (I haven't tried it, but I don't see a reason why it *wouldn't* be possible).

I'm sort of pre-empting a potential argument like this:
"the default token doesn't work"
"check our test to see a working example"
"how do I know that's not a PAT?"

I may be greatly overthinking things :laughing:

Reply:
> I'm thinking that secrets.GITHUB_TOKEN could be overridden by explicitly setting it as an actions secret (I haven't tried it, but I don't see a reason why it wouldn't be possible).

nope! all GITHUB* prefixed secrets are explicitly forbidden. if you try to set one (like a GITHUB_TOKEN with more permissive PAT perms) then it will not let you